internal/api/api.go

```diff
@@ -75,6 +78,18 @@ func (api *API) configureLogger() {
 		log.WithError(err).Warnf("failed to parse log level, falling back to %q", log.GetLevel().String())
 	}
 
+	logFormat := os.Getenv(envLogFormat)
+	if strings.EqualFold(logFormat, "json") {
```

Comment:
`strings.EqualFold()` seems like overkill here, given "json" is a constant non-UTF string.

Also, would be nice to capture strange/unexpected values, e.g:
```suggestion
	switch logFormat {
	case "json":
		log.SetFormatter(...)
	case "": // do nothing
	default:
		log.Warnf("unexpected %s value: %q", envLogFormat, logFormat)
	}
```

Reply:
It would change current behaviour. I just want to add an option to use json log format.